debian/control

```diff
@@ -0,0 +1,15 @@
+Source: gtsam
+Section: libs
+Priority: optional
+Maintainer: Bernd Pfrommer <bernd.pfrommer@gmail.com>
```

Comment:
This should be Frank?

Reply:
Following my former comment (below), yes, we should change it. It *could* be myself, since I'll continue publishing in the PPA, but let's put Frank's better, just in case this script is someday used as the base to officially upload it to Debian/Ubuntu, etc. :+1: